### ISSUE TITLE
Duplicate saved windows

### ISSUE THREAD

**Issue by jonathaneunice** [Connorhd/TabCloud#1]
When I name a tabset (say 'Zeno Project') and save it, TabCloud works nicely. When I come back later, I can open Zeno Project and work away. But if I add or delete some tabs, and save it again (to update what the cloud knows about that tabset), I get a duplicate saved tabset. A few saves later, I have half a dozen saved Zeno Project tabsets. The list of saved tabsets is not even name-sortable, making it hard to identify what is the most recent version. 

Example: http://twitpic.com/3irz8z

I thought once I named a tabset, that name would be durable, and further saves would update that tabset, not create clones or near-clones. This will be the biggest impediment to scaling up my TabCloud use.

**Comment by Connorhd** [comment]
Yeah this is something I've been meaning to sort out for a while, just not got round to it, I want to be careful not to overwrite saved tabs when they shouldn't be (for example if a user has more than one window named the same thing), and probably have an optional confirmation on overwriting to make sure users are aware its going to happen.

**Comment by jonathaneunice** [comment]
The ideal, castle-on-the-mountaintop solution would be showing the user a comparison of the saved tabset X and the newly saved tabset X' (if there are any differences). But a simple "ok to overwrite?" confirmation would be both entirely adequate and much simpler to craft. Having many identical or nearly identical tabsets named the same thing seems far less adequate.

Having persistent names for tabsets is _such_ a great approach. It exposes the clumsy obtuseness of "save every last tab under a given name" as done by most session management plugins. But if there's not a mechanism to update tabsets without proliferating them, the value of TabCloud's approach will IMO be obscured by the confusing throng.

**Comment by krzhang** [comment]
Any plan for this feature? I think it would be super high value.
